config/locales/ja.yml

```diff
@@ -274,6 +299,7 @@ ja:
     back: 戻る
     edit_profile: プロフィールを編集
     export: データのエクスポート
+    followers: フォロー承認
```

Comment:
Since original text is "Authorized followers", literal translation would be "承認済みのフォロワー", and "フォロー承認" sounds like "Approve follow requests".

However, this page is used to remove follower instances, not individual followers. so we may want to translate it like "信頼済みのインスタンス".